web/views/user.py

```diff
@@ -0,0 +1,29 @@
+from flask import Blueprint
+from flask_restful import Resource, Api
+
+from coalaip import CoalaIp
+from coalaip_bigchaindb import Plugin
+
+
+coalaip = CoalaIp(Plugin('http://localhost:9984/'))
+
+user_views = Blueprint('user_views', __name__)
+user_api = Api(user_views)
+
+
+class UserApi(Resource):
+    def post(self):
+        """API endpoint to create a new keypair for a user
+
+        Return:
+            A dict containing the verifying_key and signing_key.
+        """
+        # TODO FOR COALA IP: Return CamelCase key names
+        user = coalaip.generate_user()
```

Comment:
Not sure, it's an unrecoverable error so it's fine if the server errors out with a 500 I guess (which it should do at this point already).